shared/middleware/net_sim_api.rb

```diff
@@ -178,32 +178,75 @@ def owns_shard?(shard_id)
   #
   # POST /v3/netsim/<shard-id>/<table-name>
   #
-  # Insert a new row.
+  # Insert a new row or rows.
```

Comment:
Good call.

Is there a style guide we're using for this kind of documentation?